src/collective/collectionfilter/tests/robot/test_filterportlets_ajaxdisabled.robot

```diff
@@ -38,3 +38,9 @@ Scenario: Searching through a portlet with ajax disabled
     #   and should be 4 filter options
 
 
+Scenario: Customize search porlet text with ajax disabled
```

Comment:
needs to be rebased against the testing branch. There aren't tests specific to portlets anymore. The same test should work for tiles and portlets